prowspecs/specs.go

```diff
@@ -75,6 +75,26 @@ var knownBranches map[string]BranchSpec = map[string]BranchSpec{
 
 		skipTrivy: true,
 	},
+	"release-1.10": {
+		prowContext: &prowgen.ProwContext{
+			Branch: "release-1.10",
+
+			// Freeze test images used.
+			Image: "eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220830-c65cd19-4.2.1",
+
+			// NB: we don't use a presubmit dashboard outside of "master", currently
+			PresubmitDashboard: false,
+			PeriodicDashboard:  true,
+
+			Org:  "cert-manager",
+			Repo: "cert-manager",
+		},
+
+		primaryKubernetesVersion: "1.25",
+		otherKubernetesVersions:  []string{"1.20", "1.21", "1.22", "1.23", "1.24"},
+
+		skipTrivy: true,
```

Comment:
Question: why do we skip trivy tests here- from this comment it seems like they are skipped for versions where the trivy change wasn't backported? I assume we might want to run them against an already released version if it is the case that an updated trivy database can find new vulnerabilities in an already released version.

Reply:
We skip them because they are currently broken, I think @SgtCoDFish will take a look at fixing them when they have time

https://testgrid.k8s.io/cert-manager-periodics-master

Reply:
cool, thanks for the explanation 👍🏼